Projects/Market/Sources/LanguageAndMarketPickerView.swift

```diff
@@ -120,27 +85,27 @@ public struct LanguageAndMarketPickerView: View {
     }
 }
 
-struct LanguageAndMarketPickerView_Previews: PreviewProvider {
+struct LanguagePickerView_Previews: PreviewProvider {
     static var previews: some View {
-        LanguageAndMarketPickerView()
+        LanguagePickerView()
     }
 }
 
-class LanguageAndMarketPickerViewModel: ObservableObject {
+class LanguagePickerViewModel: ObservableObject {
     @Published var selectedLocale = Localization.Locale.currentLocale
     @Published var selectedLocaleCode: String? = Localization.Locale.currentLocale.rawValue
 
     @Published var selectedMarket: Market
```

Comment:
We don't need Market anymore. We can remove that enum also - its always going to be Sweden.